### ISSUE TITLE
Change text to speech to use the API key classes

### ISSUE THREAD

**Issue by LousyLeonard** [jsjlewis96/ai_project#13]
Change text to speech to use the API key classes.

**Comment by LousyLeonard** [comment]
See @DomHoney and @Mattihew for help.

**Comment by DomHoney** [comment]
This issue is blocked by me - branch SpeechToText needs to be finished and merged into master before this issue can be progressed.

**Comment by DomHoney** [comment]
Issue no longer blocked, Issue#13 can now pull in changes made to master branch.

**Comment by DomHoney** [comment]
This issue involved changes to the common Authentication class, which are needed in issue #20.
Opened pull request #27

**Comment by DomHoney** [comment]
Pull request merged, issue complete.
